external-crates/move/crates/move-transactional-test-runner/src/tasks.rs

```diff
@@ -43,7 +43,7 @@ pub fn taskify<Command: Debug + Parser>(filename: &Path) -> Result<Vec<TaskInput
     // checks for lines that start with //# commands
     // cutting leading/trailing whitespace
     // capturing the command text
-    let re_command_text = Regex::new(r"^\s*//#\s*(.*)\s*$").unwrap();
+    let re_command_text = Regex::new(r"^\s*//#[command(subcommand)]*(.*)\s*$").unwrap();
```

Comment:
The latest batch of CI errors was because of this -- I thought you had reverted this before?

Reply:
Ah it was in a different location before.